README.md

```diff
@@ -1,23 +1,23 @@
 <p align="center"><a href='http://reactide.io/'><img alt="reactide" src="http://reactide.io/images/reactide-header.png" height="60%" width="60%"></a></p>
 
 ### Reactide is the first dedicated IDE for React web application development
-Reactide is a cross-platform desktop application that offers a custom simulator, making build-tool and server configuration unnecessary. Reactide brings development back to the days where opening a single file instantly renders the project in the browser. With Reactide, developers can achieve the same simplicity with a single React JSX file while still utilizing the power of React.
+Reactide is a cross-platform desktop application that offers a custom simulator, making build-tool and server configuration unnecessary. Reactide brings development back to the days where opening a single file instantly renders the project in the browser. Developers can achieve the same simplicity with a single React JSX file while still utilizing the power of React.
 
 #### Reactide is in active development. Please follow this repo for contribution guidelines and our development road map.
 
-## 
+##
 <p align="center">
   <img alt="Reactide Screenshot" src="http://reactide.io.s3-website-us-west-1.amazonaws.com/images/reactide-screenshot2.png">
 </p>
 
 ## Get right to coding
-Reactide runs an integrated Node server and custom browser simulator, which eliminates the need to configure servers, build-tools, and even offers hot module reloading right out of the box. Projects developed in Reactide are build-tool agnostic. As projects evolve, the developer only needs to add necessary dependencies without having to make decisions before coding has even started.
+Reactide runs an integrated Node server and custom browser simulator. Configuring servers and build-tools are eliminated, and hot module reloading is offered right out of the box. Projects developed in Reactide are build-tool agnostic. As projects evolve, the developer only needs to add necessary dependencies without having to make decisions before coding has started.
```

Comment:
Just to be clear, I was suggesting:

> Configuring servers and build-tools is eliminated

instead of are eliminated. Just feels weird to me read that. But hey, I may be nitpicking. Good job nonetheless. 👍